shared/AppInsightsCore/src/JavaScriptSDK/AppInsightsCore.ts

```diff
@@ -791,6 +789,9 @@ export class AppInsightsCore<CfgType extends IConfiguration = IConfiguration> im
                 _configExtensions = [];
                 _channelConfig = null;
                 _channels = null;
+                let cfgChannels = _self.config.channels || [];
```

Comment:
This is not needed -- I tried "populating" the channels with the resolved values and it ended up duplicating them..